docs/sources/alerting/manage-notifications/view-alert-groups.md

```diff
@@ -4,49 +4,103 @@ aliases:
   - ../../alerting/alert-groups/filter-alerts/ # /docs/grafana/<GRAFANA_VERSION>/alerting/alert-groups/filter-alerts/
   - ../../alerting/alert-groups/view-alert-grouping/ # /docs/grafana/<GRAFANA_VERSION>/alerting/alert-groups/view-alert-grouping/
   - ../../alerting/unified-alerting/alert-groups/ # /docs/grafana/<GRAFANA_VERSION>/alerting/unified-alerting/alert-groups/
+  - ../../alerting/manage-notifications/view-notification-errors/ # /docs/grafana/<GRAFANA_VERSION>/alerting/manage-notifications/view-notification-errors/
 canonical: https://grafana.com/docs/grafana/latest/alerting/manage-notifications/view-alert-groups/
-description: Alert groups
+description: The Groups view lists grouped alerts that are actively triggering notifications.
 keywords:
   - grafana
   - alerting
   - alerts
+  - errors
+  - notifications
   - groups
 labels:
   products:
     - cloud
     - enterprise
     - oss
-title: View and filter by alert groups
+title: View the status of notifications
 weight: 800
+refs:
+  alertmanager:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/set-up/configure-alertmanager/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/set-up/configure-alertmanager/
+  grouping:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/notifications/group-alert-notifications/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/notifications/group-alert-notifications/
 ---
 
-# View and filter by alert groups
+# View the status of notifications
 
-Alert groups show grouped alerts from an Alertmanager instance. By default, alert rules are grouped by the label keys for the default policy in notification policies. Grouping common alert rules into a single alert group prevents duplicate alert rules from being fired.
+The Groups view page lists grouped alerts that are actively triggering notifications.
 
-You can view alert groups and also filter for alert rules that match specific criteria.
+By default, Grafana Alerting groups similar firing alerts (or alert instances) to prevent notification overload. For details on how notification grouping works, refer to [Group alert notifications](ref:grouping).
 
-## View alert groups
+In the Groups view, you can see alert groups, check the state of their notifications, and also filter for alert instances that match specific criteria. This view is useful for debugging and verifying your grouping of notification policies.
```

Comment:
nit: perhaps disambiguate with "grouping settings of notification policies" since policies can't be grouped :D

Reply:
addressed by https://github.com/grafana/grafana/pull/89461/commits/0d2e9d639b9228713e121a44869e5145e88867eb